### PR TITLE
send mail when start, finish exam

### PR DESCRIPTION
**Pull request by dat-hedspi**

### CODE REVIEW

File: app/workers/user_worker.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    exam = Exam.find_by id: exam_id
+    UsersMailer.start_to_exam(user, exam).deliver
+  end
+
+  def send_email_when_finish_exam user_id, exam_id
+    user = User.find_by id: user_id
+    exam = Exam.find_by id: exam_id
+    UsersMailer.finish_to_exam(user, exam).deliver
+  end
+end
```
Comment (nguyenngocha) [inline app/workers/user_worker.rb]: end file

File: config/locales/en.yml [hunk 2]
```diff
@@ -24,21 +24,21 @@ en:
     contain: This form has
     error: error
   exam:
+    checked: View
     create:
       error: Create Exam Error
       success: Create Exam Success
+    finished: Exam has finished
+    init: Start
+    result: 'Score of this exam: %{score}'
```
Comment (nguyenngocha) [inline config/locales/en.yml]: double quote

File: config/environments/development.rb [hunk 3]
```diff
@@ -30,7 +32,18 @@
...
   # yet still be able to expire them through the digest params.
   config.assets.digest = true
-
+  config.action_mailer.raise_delivery_errors = true
+  # config.action_mailer.delivery_method = :test
+  config.action_mailer.smtp_settings = {
+    address: "smtp.gmail.com",
+    port: "587",
+    domain: "localhost",
+    user_name: "dat.hedspi@gmail.com",
```
Comment (nguyenngocha) [inline config/environments/development.rb]: dung bien ENV

**Comment by tiennh1995** [comment]
ok

**Comment by nguyenngocha** [comment]
OK

**Pull request merged**